jsk_topic_tools/CMakeLists.txt

```diff
@@ -87,7 +87,9 @@ add_library(jsk_topic_tools SHARED
 add_dependencies(jsk_topic_tools ${PROJECT_NAME}_gencpp)
 target_link_libraries(jsk_topic_tools ${catkin_LIBRARIES})
 
-if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
+# ``add_rostest`` should work with latest catkin on even if indigo
+# See: https://github.com/jsk-ros-pkg/jsk_common/pull/1178#discussion_r41753604
+# if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
```

Comment:
I updated.

2015年10月19日月曜日、Ryohei Ueda<notifications@github.com>さんは書きました:

> In jsk_topic_tools/CMakeLists.txt
> <https://github.com/jsk-ros-pkg/jsk_common/pull/1190#discussion_r42348952>
> :
>
> > @@ -87,7 +87,9 @@ add_library(jsk_topic_tools SHARED
> >  add_dependencies(jsk_topic_tools ${PROJECT_NAME}_gencpp)
> >  target_link_libraries(jsk_topic_tools ${catkin_LIBRARIES})
> >
> > -if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
> > +# ``add_rostest`` should work with latest catkin on even if indigo
> > +# See: https://github.com/jsk-ros-pkg/jsk_common/pull/1178#discussion_r41753604
> > +# if (NOT $ENV{ROS_DISTRO} STREQUAL "indigo")
>
> remove it instead of comment out
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_common/pull/1190/files#r42348952>.
>


-- 
和田 健太郎 / Kentaro Wada
http://wkentaro.com